src/current/_includes/v23.2/sql/disallow-full-table-scans.md

```diff
@@ -0,0 +1,12 @@
+- At the cluster level, set `disallow_full_table_scans` for some or all users/roles. For example:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER ROLE ALL SET disallow_full_table_scans = true;
+    ~~~
+
+- At the application level, add `disallow_full_table_scans` to the connection string using the [`options` parameter]({% link {{page.version.version}}/connection-parameters.md %}#additional-connection-parameters).
+
+If you disable full scans, you can set the [`large_full_scan_rows` session variable]({% link {{ page.version.version }}/set-vars.md %}#large-full-scan-rows) to specify the maximum table size allowed for a full scan. If no alternative plan is possible, the optimizer will return an error.
+
+If you disable full scans, and you provide an [index hint]({% link {{ page.version.version }}/indexes.md %}#selection), the optimizer will try to avoid a full scan while also respecting the index hint. If this is not possible, the optimizer will return an error. If you do not provide an index hint, the optimizer will return an error, the full scan will be logged, and the `sql.guardrails.full_scan_rejected.count` [metric]({% link {{ page.version.version }}/ui-overview-dashboard.md %}) will be updated.
```

Comment:
> If you do not provide an index hint, the optimizer will return an error ...

Shouldn't this say something like "If you do not provide an index hint, but it's still not possible to avoid a full scan, the optimizer will return an error..."